### DISCUSSION TITLE
Request a custom firmware - v2.10.x

### DISCUSSION THREAD

**Discussion by pfeerick** [EdgeTX/edgetx#4992] [General]
Ask here if you want/need specific options that are not included in the stock builds, and it is not possible to build via the CloudBuild tab on [EdgeTX Buddy](https://buddy.edgetx.org/). . Builds will be uploaded to [here (same link as 'By request' below)](http://files.portabledev.xyz/d/e20976a0cff541679899/), so check it out before making a request just in case there is already a custom build with your chosen options...

The Language builds are automated builds based on the 2.10 branch. 

*Please remember to specify your transmitter when requesting a build* ;)

- [Language builds](https://github.com/pfeerick/lang-firmwares/releases/tag/v2.10.0) - zip file per transmitter containing all languages built, with some additional builds for enabling bluetooth or lua mixer scripts for some radios. 
- [By request](http://files.portabledev.xyz/d/e20976a0cff541679899/)

**Comment by chrisu365** [comment]
Hello, i have filed a bug: #5332 but this could probably take a while..
Maybe someone can help me and show me an option to compile a firmware for my Xlite V1 (mode 2) that uses the trimswitches by default on the right stick and when pressing the SHIFT-Button on the left stick ?
I tried some variations by modifiying edgetx/radio/util/hw_defs/generator.py edgetx/radio/util/hw_defs/hal_keys.py and edgetx/radio/src/targets/taranis/hal.h (and some more files)  compiling on gitpod , but my results did not work :(
Best regards, chris

**Comment by P-I-Engineer** [comment]
could you please add the radiomaster pocket to the firmware with custom lua mixer scripts enabled.

thank you

**Comment by caixiaohai0** [comment]
Hello, can you add the built-in multi-protocol module to X9lites? I found that this option is not supported since V2.10.3. Thank you very much, lucky every day

**Reply by pfeerick** [reply]
Can I just get clarification on this? X9LiteS was never sold in a configuration that has internal MPM AFAIK, so is this a modified radio? That would explain why the option is missing, as for normal builds it is a completely incorrect/invalid option ;)

**Reply by caixiaohai0** [reply]
> Can I just get clarification on this? X9LiteS was never sold in a configuration that has internal MPM AFAIK, so is this a modified radio? That would explain why the option is missing, as for normal builds it is a completely incorrect/invalid option ;)

Yes, because I changed the built-in module to multi protocol, could you help me compile a firmware? I am grateful

**Comment by WillyZehnder** [comment]


**Reply by WillyZehnder** [reply]
I think I've got it.
I have to specify the subversion in the language builds and there I can get the version I need:
https://github.com/pfeerick/lang-firmwares/releases/tag/v2.10.6

But btw, I still don't understand why the Gitpod build of v2.10.6 for TX16S is producing an error.
